### DISCUSSION TITLE
How to change database

### DISCUSSION THREAD

**Discussion by PasDoe** [1Remote/1Remote#379] [Q&A]
**Describe the bug**
Is there a way to change the location of the database? I've installed 1Remote via winget (winget install premotem) and didn't find a way to do this. The Appdata mode is inforced.

**Desktop (please complete the following information):**
 - OS: Win10 Pro 22H2
 - PRemoteM Version: 0.7.2.6

**Comment by VShawn** [comment]
You can follow steps:

1. find your database and copy it to new location
2. go to setting page and click `select` button in picture below, find you database in you new location.

![image](https://user-images.githubusercontent.com/10143738/223448532-d834c279-447e-464d-a1f6-5047f6d40699.png)
